irc/client.go

```diff
@@ -269,6 +269,7 @@ type WhoWas struct {
 	username       string
 	hostname       string
 	realname       string
+	realIP         net.IP
```

Comment:
Maybe this shouldn't be called "realIP" since it could be the proxied IP? What should it be called instead?

Reply:
I think `ip` is fine actually since it's the value that would have been returned by `(*Client).IP()`

Reply:
Renamed to ip.